src/main/target/OMNIBUSF4/target.c

```diff
@@ -39,10 +39,19 @@ const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM8,  CH3, PC8,  TIM_USE_PWM,                 TIMER_OUTPUT_NONE,     0), // S5_IN
     DEF_TIM(TIM8,  CH4, PC9,  TIM_USE_PWM,                 TIMER_OUTPUT_NONE,     0), // S6_IN
 
-    DEF_TIM(TIM3,  CH3, PB0,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 0), // S1_OUT D1_ST7
-    DEF_TIM(TIM3,  CH4, PB1,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 0), // S2_OUT D1_ST2
-    DEF_TIM(TIM2,  CH4, PA3,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 1), // S3_OUT D1_ST6
-    DEF_TIM(TIM2,  CH3, PA2,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 0), // S4_OUT D1_ST1
+    // Combination of N_CHANNEL and INVERTED
+#if 0
+    DEF_TIM(TIM1,  CH2N,PB0,  TIM_USE_MOTOR,               TIMER_OUTPUT_N_CHANNEL, 1), // S1_OUT D(2,2)
+    DEF_TIM(TIM1,  CH3N,PB1,  TIM_USE_MOTOR,               TIMER_OUTPUT_N_CHANNEL|TIMER_OUTPUT_INVERTED, 0), // S2_OUT D(2,6)
+    DEF_TIM(TIM2,  CH4, PA3,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 1), // S3_OUT D(1,6)
+    DEF_TIM(TIM2,  CH3, PA2,  TIM_USE_MOTOR,               TIMER_OUTPUT_INVERTED, 0), // S4_OUT D(1,1)
+#else
+    // For flight test
+    DEF_TIM(TIM1,  CH2N,PB0,  TIM_USE_MOTOR,               TIMER_OUTPUT_N_CHANNEL, 1), // S1_OUT D(2,2)
```

Comment:
`DEF_TIM_OUTPUT` infers `TIMER_OUTPUT_N_CHANNEL` from channel specification, there is no need to specify it explicitly (I'all fix the macro if it does not work)